addons/website_event/views/event_templates.xml

```diff
@@ -607,6 +607,7 @@
                         <thead>
                             <tr>
                                 <th>Reference</th>
+                                <!-- Note: we override this on website_event_sale to add the ticket here. -->
```

Comment:
No need to describe all the areas that are gonna be overridden on another module.